client/infinispan-key-value-store-hotrod/pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+   <modelVersion>4.0.0</modelVersion>
+   <parent>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-parent</artifactId>
+      <version>10.0.0-SNAPSHOT</version>
+      <relativePath>../../pom.xml</relativePath>
+   </parent>
+
+   <artifactId>infinispan-key-value-store-hotrod</artifactId>
```

Comment:
I'd leave `hotrod` out of the artifact name. True, the current client artifact contains it but that was created 9 years ago. This should be seen as the Java client for talking to a key value store (e.g. `infinispan-key-value-store-client`. In the future we might to say use grpc or something else instead of Hot Rod but quicker evolution of the clients, operations...etc.

Reply:
ok